packages/fxa-event-broker/lib/serviceNotifications.ts

```diff
@@ -55,7 +55,10 @@ const SUBSCRIPTION_UPDATE_SCHEMA = joi
       .array()
       .items(joi.string())
       .required(),
-    productName: joi.string().required(),
+    productId: joi.string().optional(),
+    // TODO: productName is the legacy name for productId, remove it
+    //       in due course then make productId required again.
+    productName: joi.string().optional(),
```

Comment:
(...and necessitates these knock-on changes)

@bbangert, I figured the correct first step in the event broker was to accept both variants as properties, then remove `productName` and make `productId` required later.

Reply:
At least one of these is necessary, I believe there's a joi OneOf thing.

Reply:
Hmm, guess the typesafe-joi doesn't work with `object.or` right now, so this is fine.